samples/Nancy.Swagger.Annotations.Demo/Startup.cs

```diff
@@ -29,7 +31,33 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                 app.UseDeveloperExceptionPage();
             }
 
-            app.UseOwin(x => x.UseNancy());
```

Comment:
Maybe I'm misunderstanding this code, but aren't you just telling Nancy to use a specific bootstrapper instead of using the default one?
When you start up the server, it would be using your StandaloneOutputBootstrapper, which would be fine because it extends the Demo Bootstrapper, but it seems unnecessary to me. 
I thought you idea was to have a separate project that references the nancy service project. Why don't you make a demo project to do that?